### PR TITLE
fix: outline function enabled for keyboard navigation in reports

### PR DESCRIPTION
**Pull request by GusBaamonde**
Related to: https://github.com/FromDoppler/Doppler-UI-System/pull/217

**No keyboard navigation**
outline-disabled

**With keyboard navigation**
outline-enabled

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <link
     rel="stylesheet"
     type="text/css"
-    href="https://cdn.fromdoppler.com/doppler-ui-library/v3.25.0/css/styles.css"
+    href="https://cdn.fromdoppler.com/doppler-ui-library/build667/css/styles.css"
```
Comment (andresmoschini) [inline public/index.html]: Remember adding change it to the final version before merging.

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js [hunk 2]
```diff
@@ -9,11 +9,11 @@ const HeaderUserMenu = ({ user }) => {
 
   return (
     <div>
-      <span className="user-menu--open">
+      <a href="#" className="user-menu--open">
```
Comment (andresmoschini) [inline src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: If I am not wrong, this is not a good practice: if it is not a real link, it should be a button.

**Comment by andresmoschini** [comment]
❤️

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js (hunk 2 above)
Comment (GusBaamonde) [reply src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: Yes!!! Thanks!

**Comment by cbernat** [comment]
👍

**Pull request merged**